partcad/src/partcad/healthcheck/available_disk_space.py

```
@@ -0,0 +1,48 @@
+        try:
+            path = user_config.internal_state_dir
+
+            _, _, free = shutil.disk_usage(path)
+
+            min_space_bytes = self.min_space * 1_000_000_000
+
+            if free < min_space_bytes:
+                self.findings.append(
+                    f"Insufficient disk space. Need at least {self.min_space} GB free in {path}. Currently, only {free // (1024 * 1024 * 1024)} GB is available."
+                )
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)